src/client/datascience/interactive-window/nativeInteractiveWindow.ts

```diff
@@ -591,11 +591,19 @@ export class NativeInteractiveWindow implements IInteractiveWindowLoadable {
         // Ensure we have a controller to execute code against
         // and a NotebookDocument to add the NotebookCell to
         const notebookDocument = this.notebookDocument;
+        if (!this.notebookController) {
```

Comment:
we don't need to call this anymore considering we have passed the id to the `interactive.open` command. Keeping this will trigger the kernel picker sometimes, which we don't want to see.

Reply:
@rebornix If the id is undefined, e.g. there is no selected Python interpreter, and there is no cached association in VS Code, I believe we would still need to prompt the user to select a kernel for the first time.

Reply:
Is this state here ok if the users just hits escape in the kernel picker and doesn't pick anything?

Reply:
Yeah, we simply won't execute anything until a controller is selected for this window.